oshi-core/src/main/java/oshi/hardware/platform/mac/MacDisks.java

```diff
@@ -211,161 +382,7 @@
 
                 // Now look up the device using the BSD Name to get its
```

Comment:
this comment belongs with the moved code :)

Reply:
True!